crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -63,13 +67,20 @@ fn initialize<B: Board>(mut call: SchedulerCall<B, api::initialize::Sig>) {
     let result = try {
         let context = match convert_hash_algorithm::<B>(*algorithm)?? {
             #[cfg(feature = "board-api-crypto-sha256")]
-            Algorithm::Sha256 => HashContext::Sha256(board::crypto::Sha256::<B>::default()),
+            Algorithm::Sha256 => {
+                HashApi::<board::crypto::Sha256<B>>::new().map(HashContext::Sha256)
```

Comment:
nit: the type can be inferred (same for the 3 other occurrences below)
```suggestion
                board::crypto::HashApi::new().map(HashContext::Sha256)
```
And by fully qualifying the HashApi type, no need to import anything.